src/components/widget/legend/component.js

```diff
@@ -13,14 +13,22 @@ const Legend = ({ groups, direction }) => (
           {groups[g].map(item => (
             <li
               key={`item-${item.color}`}
-              className={styles.widget_legend_list_item}
+              className={classnames(
+                styles.widget_legend_list_item,
+                { [styles.net]: true }
```

Comment:
If this is true object is not needed.

You don't even need classnames `className={styles.widget_legend_list_item+' '+styles.net}` or ``{`${styles.widget_legend_list_item} ${styles.net}`}``but if you want to use it is ok.
```suggestion
                styles.net
```